influxdb/__init__.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 from .client import InfluxDBClient
 from .dataframe_client import DataFrameClient
+from .helper import SeriesHelper
 
 
 __all__ = [
     'InfluxDBClient',
     'DataFrameClient',
+    'SeriesHelper',
 ]
 
 
-__version__ = '0.1.13'
+__version__ = '0.1.14'
```

Comment:
No, I will bump the version myself.
I could release 0.1.14 before I merge your PR.